aten/src/ATen/native/TensorCompare.cpp

```diff
@@ -106,6 +108,50 @@ Tensor isinf(const Tensor &self) {
   });
 }
 
+static inline TensorIterator build_is_infinity_op_iterator(Tensor& result, const Tensor& self) {
+  return at::TensorIteratorConfig()
+    .check_all_same_dtype(false)
+    .set_check_mem_overlap(true)
+    .add_output(result)
+    .add_input(self)
+    .cast_common_dtype_to_outputs(true)
+    .build();
+}
+
+Tensor isposinf(const Tensor &self) {
+  Tensor result = at::empty_like(self, at::kBool, at::MemoryFormat::Preserve);
+  at::isposinf_out(result, self);
+  return result;
+}
+
+Tensor& isposinf_(Tensor &self) {
+  return at::isposinf_out(self, self);
+}
+
+Tensor& isposinf_out(Tensor& result, const Tensor& self) {
+  result.resize_(self.sizes());
+  auto iter = build_is_infinity_op_iterator(result, self);
+  isposinf_stub(iter.device_type(), iter);
+  return result;
+}
+
+Tensor isneginf(const Tensor &self) {
```

Comment:
See parallel comments on isposinf

Reply:
The `isneginf` docs have been updated according to the comments of `isposinf` docs.